concent_api/gatekeeper/views.py

```diff
@@ -35,7 +35,7 @@
 @require_POST
 def upload(request):
     logger.debug("Upload request received.")
-    if request.content_type != 'application/x-www-form-urlencoded':
+    if request.content_type != 'multipart/form-data':
```

Comment:
I think this should like this `if request.content_type == 'multipart/form-data'` or `if request.content_type != 'application/octet-stream'`

Reply:
Done